backend/Application/Api/GraphQL/Extensions/ScalarTypes/BigIntegerScalarType.cs

```diff
@@ -0,0 +1,94 @@
+using System.Numerics;
+using HotChocolate.Language;
+using HotChocolate.Types;
+
+namespace Application.Api.GraphQL.Extensions.ScalarTypes
```

Comment:
I am not sure I understand the purpose here. Can you please explain what this is doing for future maintainers.

Reply:
Have added it to doc comment.

Reply:
Looks good.